build/x86_64_amzn-2016.09/Dockerfile

```diff
@@ -22,9 +22,11 @@ RUN yum -y install epel-release \
         perl-devel \
         perl-ExtUtils-Embed \
         pkgconfig \
+        postgresql-devel \
```

Comment:
Just curious: How did Varnish and Postgresql work before this change???

Reply:
We don't build the postgresql and varnish plugins in the 6.x.x versions (only in the legacy 5.5.2 versions), and we are still using the Dockerfiles from the `stackdriver-agent-5.5.2` branch. This will allow us to switch all build containers to the master branch.